src/panels/config/entity_registry/dialog-entity-registry-detail.ts

```diff
@@ -63,6 +63,11 @@ class DialogEntityRegistryDetail extends LitElement {
       computeDomain(this._params.entry.entity_id);
 
     return html`
+      <style include="ha-style-dialog">
+        :host {
+          --paper-font-title_-_white-space: normal;
+        }
```

Comment:
Just add this to `static get styles()` instead of in `render`

Reply:
Done!